snowshu/core/graph.py

```diff
@@ -198,31 +215,33 @@ def _build_sum_patterns_from_configs(
             for lower_level in upper_level.relationships.bidirectional + upper_level.relationships.directional
         ]
 
-        all_patterns = approved_default_patterns + approved_specified_patterns +approved_second_level_specified_patterns
+        all_patterns = approved_default_patterns + \
+            approved_specified_patterns + approved_second_level_specified_patterns
         logger.debug(f'All config primary patterns: {all_patterns}')
         return all_patterns
 
-    def _filter_relations(self, full_catalog: iter,
+    @staticmethod
+    def _filter_relations(full_catalog: iter,
                           patterns: dict) -> Set[Relation]:
         """applies patterns to the full catalog to build the filtered relation
         set."""
 
-        return set([filtered_relation for filtered_relation in \
-            set(filter(lambda rel : at_least_one_full_pattern_match(rel,patterns), full_catalog))])
-    
-    def _set_globals_for_node(self,relation:Relation,configs:Configuration)->Relation:
+        return set(filter(lambda rel: at_least_one_full_pattern_match(rel, patterns),
```

Comment:
Simplifying list comprehension like this is one of the common fixes.

Reply:
Other Ref 
* https://github.com/Health-Union/snowshu/pull/57/files#diff-590ce9cf5bff2b2c77e21415de71bcfc7d2f22cad83e42bf424160da3ce2f9a3R177